f/missionConditions/f_setMissionConditions.sqf

```diff
@@ -14,6 +14,12 @@ if (isServer) then {
 		f_var_conditions_params = [date] + _weatherParams;
 	};
 
+	// COLD BREATH
+	// Cold breath particle spawner for if it's snowing, or in low ambient temperatures
+	if (f_param_weather == 9 || f_param_weather == 10 || (ambientTemperature select 0) < 8) then {
+		remoteExec ["f_fnc_coldBreath",0,true];
```

Comment:
```suggestion
		remoteExec ["f_fnc_coldBreath",-2,true];
```
Minor: Maybe exclude the server since this is a client-side effect?

Reply:
Should be `[0,-2] select isDedicated` otherwise it will cause trouble in local-hosted MP

Reply:
Good point. Ignore me, this isn't worth changing since the function has a `hasInterface` check anyway